Maestro/subscriptions.json

```diff
@@ -1,11 +1,17 @@
 {
   "actions": {
-    // A build definition capable of running any .ps1 script in the CoreFX repo
+    // A build definition capable of running any .ps1 script in the CoreFX repo master branch
     "corefx-general": {
       "vsoInstance": "devdiv.visualstudio.com",
       "vsoProject": "DevDiv",
       "buildDefinitionId": 1084
     },
+    // A build definition capable of running any .ps1 script in the CoreFX repo release branch
```

Comment:
can you say release/1.0.0 branch?  To make it more specific.

Like I said offline, with a new Maestro feature, you will be able to specify the branch without having to create a new VSO build definition.  I'll try to get this today or tomorrow.